### PR TITLE
Enhance filtering of messages

### PR DESCRIPTION
**Pull request by BjrInt**
## Problem

It is getting hard to look for specific messages with the current UI.

## Proposed solution

- [x] Use an auto-completed text-input for channel selection

### CODE REVIEW

**Comment by BjrInt** [comment]
The feature proposal looks like this:

[Screencast from 2022-12-07 12-02-33.webm](https://user-images.githubusercontent.com/26065817/206162608-03f188a9-b062-4cd5-850b-2ed2d1f4410a.webm)

**Comment by cpascariello** [comment]
> The feature proposal looks like this:
> 
>  Screencast.from.2022-12-07.12-02-33.webm

This is great!
Now we only need to find a way to unclutter the list.

File: src/views/Messages.vue [hunk 1]
```diff
@@ -89,27 +83,33 @@ export default {
...
-  async created() {
+  async created () {
     await this.refresh()
   }
 }
 </script>
+
+<style>
+:root{
+  --vs-font-size: unset;
```
Comment (hoh) [inline src/views/Messages.vue:113]: What is this for ?

**Comment by cpascariello** [comment]
> > The feature proposal looks like this:
> > Screencast.from.2022-12-07.12-02-33.webm
> 
> This is great! Now we only need to find a way to unclutter the list.

Maybe an idea is to add a search field in the dropdown to filter the list.

File: src/views/Messages.vue (hunk 1 above)
Comment (BjrInt) [reply src/views/Messages.vue:113]: The component I use (vue-select), uses [css variables to style the component](https://vue-select.org/guide/css.html#dark-mode-example), `--vs-font-size` is set to `1rem` by default which looks off with rest of the UI. 

Unsetting it, means that we use the default value provided by the parent.

**Comment by BjrInt** [comment]
> Maybe an idea is to add a search field in the dropdown to filter the list.

It's already there (although maybe not showcased very well in the video) :wink:

File: src/views/Messages.vue (hunk 1 above)
Comment (hoh) [reply src/views/Messages.vue:113]: Would it make sense to add this explanation as a comment in this code ? I doubt we would remember later...

Comment (BjrInt) [reply src/views/Messages.vue:117]: Done

**Comment by cpascariello** [comment]
Oh nice!

Sent from Outlook for iOS<https://aka.ms/o0ukef>
________________________________

**Pull request merged**